carl/context_encoders/configs/base.yaml

```diff
@@ -0,0 +1,8 @@
+encoder.
+  outdir: '/home/mohan/git/CARL/tmp/AE_stuff/AE_Test'
+  context_db: '/home/mohan/git/CARL/tmp/AE_database/Pendulum/new_60k.npy'
```

Comment:
where is the file which generated the dataset? this is also important such that we can regenerate the training set and then provide a fitting path.
- [ ] also, please remove this path because the data is not in the repo. you could add a check wether `context_db` is `None` and if so, raise a ValueError stating how to generate the context training set.
- [ ] Could you also rename `context_db` to `context_dataset_train` or sth similar?

Reply:
Added an example database and an additional script to generate it from a contexts_train.json file. Put instructions as a value error in the HyperBand code